flink-clients/src/test/java/org/apache/flink/client/deployment/application/ApplicationDispatcherBootstrapTest.java

```diff
@@ -348,6 +350,10 @@ void testApplicationIsStoppedWhenStoppingBootstrap() throws Exception {
 
         bootstrap.stop();
 
+        // the JobStatusFuture needs to be completed after stopping the bootstrap process to ensure
+        // that the process doesn't finish before explicitly stopping it
```

Comment:
I have no idea what "it" refers to.

Reply:
You got me on that one. I mixed up the `applicationExecutionTask` and the `applicationCompletionFuture` and, therefore, also came up with a wrong comment. Initially, I was puzzled why it worked anyway. I updated the comment to reflect the actual reason why this works. PTAL